types/got/index.d.ts

```diff
@@ -98,15 +98,19 @@ declare namespace got {
         json?: boolean;
     }
 
-    type GotOptions<E extends string | null> = http.RequestOptions & {
+    interface TimoutRequestOptions extends http.RequestOptions {
```

Comment:
Is there a reason why this wasn't just added to `GotOptions`, instead of creating a new type and extending?